assets/src/hooks/useChannel.ts

```diff
@@ -16,13 +16,15 @@ export const useChannel = <T>({
   parser,
   loadingState,
   closeAfterFirstRead,
+  dependencies,
```

Comment:
I think this may be questionable -- further comment about the possibility of removing this change below